plugin/internal/command.go

```diff
@@ -0,0 +1,148 @@
+package internal
```

Comment:
I don't want to export `internal.Command` because it was created to solve a specific use-case. If this makes sense, then I think we should move the core plugins over to a `plugin/core` folder so a newer Go user doesn't think this is a core plugin named `internal` (and it also makes sense to place them in their own directory).

Reply:
I think the module description can handle that pretty well. Which we should add here.

Reply:
I can add a module description.

What's the reason for not moving the core plugins over to `plugin/core` ?

Reply:
I don't think it'll actually improve clarity. Will https://godoc.org/github.com/puppetlabs/wash/plugin look any different? The table at https://godoc.org/github.com/puppetlabs/wash seems pretty straightforward to me, and we use internal other places so that seems like a clear pattern.

Reply:
Hmm, think you're probably right. `internal` is a documented Go pattern for modular code you don't want to export.

Ok, I'll just add a module description.